scripts/core/editor3/tests/Editor3Component.spec.jsx

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import {shallow} from 'enzyme';
+import {Editor3Component} from '../components/Editor3';
+import createEditorStore from '../store';
+import {EditorState, ContentState} from 'draft-js';
+import ng from 'core/services/ng';
+import reducer from '../reducers';
+
+describe('Editor3Component', () => {
+    it('check if toolbar is not showed', () => {
+        const wrapper = shallow(<Editor3Component showToolbar={ false } onChange={(x) => x} />);
+
+        expect(wrapper.find('DraftEditor').length).toBe(1);
+        expect(wrapper.find('Toolbar').length).toBe(0);
+    });
+});
+
+describe('Editor3.createEditorStore', () => {
+    it('should initialize with correct values', inject(($q) => {
+        const setLanguage = jasmine.createSpy(),
+            getDict = jasmine.createSpy().and.callFake(() => $q.when(null)),
+            isCorrectWord = jasmine.createSpy();
+
+        ng.mock('spellcheck', {setLanguage, getDict, isCorrectWord});
```

Comment:
given that it's based on `$injector`, you could use probably:
```
beforeEach(inject(($provide) => {
    $provide.service('spellchecker', ($q) => ({
        setLanguage: jasmine.createSpy(),
        getDict: jasmine.createSpy().and.returnValue($q.when(null)),
        isCorrectWord: jasmine.createSpy()
    }));
}));
```
so you wouldn't need to add that mock func to `ng` and you don't have to clean it after